### PR TITLE
FHAC-699:Publish AuditQueryLogUnsecured Service

### PR DESCRIPTION
**Pull request by alameluchidambaram**
1.Publish Unsecured endpoint for AuditQueryLog service.

### CODE REVIEW

File: Product/Production/Gateway/CONNECTGatewayWeb/src/main/webapp/WEB-INF/cxf-servlet.xml [hunk 1]
```diff
@@ -25,17 +25,17 @@ http://cxf.apache.org/core http://cxf.apache.org/schemas/core.xsd ">
         </cxf:features>
     </cxf:bus>
 
-    <!-- Audit Services -->
+    <!-- Audit Reposittory Services -->
```
Comment (christophermay07) [inline Product/Production/Gateway/CONNECTGatewayWeb/src/main/webapp/WEB-INF/cxf-servlet.xml]: Typo (use Repository)

**Comment by christophermay07** [comment]
I've reviewed this PR, endorsing for merge without further comment.

**Comment by alameluchidambaram** [comment]
@christophermay07 @jsmith2 @mhpnguyen

**Comment by mhpnguyen** [comment]
Should we have javadoc for all methods?

**Comment by mhpnguyen** [comment]
Endorsing for merging.

**Pull request merged**